package-system/vulkan-validationlayers/build_config.json

```diff
@@ -0,0 +1,59 @@
+{
+    "git_url":"https://github.com/KhronosGroup/Vulkan-ValidationLayers.git",
+    "git_tag":"v1.2.198",
+    "package_name":"vulkan-validationlayers",
+    "package_version":"1.2.198",
+    "package_url":"https://vulkan.lunarg.com/",
+    "package_license":"Apache-2.0",
+    "package_license_file":"LICENSE.txt",
+    "cmake_find_source":"Findvulkan-validationlayers.cmake",
+    "cmake_find_target":"Findvulkan-validationlayers.cmake",
+    "cmake_src_subfolder": "vulkan-validation",
+    "cmake_build_args" : [ "-j", "8"],
+    "cmake_generate_args_common": [],
+    "cmake_build_args_common": [
+        "--parallel"
+    ],
+    "Platforms":{
+        "Windows":{
+            "Windows":{   
+				"custom_build_cmd" : [
+                    "build_vulkan_validation_windows.cmd"
+                ],
+                "custom_install_cmd": [
+                    "install_vulkan_validation_windows.cmd"
+                ]
+            },
+			"Android":{
+                "custom_build_cmd" : [
+                ],
+                "custom_install_cmd": [
+                ]
+            }
+        },
+		"Darwin":{
+            "Mac":{
+                "custom_build_cmd": [
+                 ],
+                 "custom_install_cmd": [
+                 ]
+            },
+            "iOS":{
+                "custom_build_cmd": [
+                 ],
+                 "custom_install_cmd": [
+                 ]
+             }
+        },
+        "Linux":{
+            "Linux":{
+                "custom_build_cmd": [
+                ],
+                "custom_install_cmd": [
+                ],
+                "custom_test_cmd" : [
+               ]
+            }
```

Comment:
There are no scripts to build Linux yet, so this PR is just for Windows right?

Reply:
Linux and Android are just stubs now since they use vulkan on Atom as well. ly_add_target() in the Atom.RHI.Vulkan cmake will have 3rdParty::vulkan-validation-layers.

Reply:
If you intend on creating 'stub' packages, then i'd suggest not sharing the same 'Findvulkan-validationlayers.cmake' as the source file. Instead create a 'Findvulkan-validationlayers.cmake.stub' as the 'cmake_find_source' for those platforms, and have that find just declare the empty library, and add comments there saying that it is not supported yet. 

Or even simpler, use PAL to only add a dependency to atom on windows for this package, and don't declare anything for platforms other than windows. So you would only need to push 1 package instead of stubs.

Reply:
Added stubs for linux and android.